cmd/cli/version.go

```diff
@@ -106,7 +106,7 @@ func (v *versionCmd) getMeshVersion() (*remoteVersionInfo, error) {
 		return nil, err
 	}
 	if len(controllerPods.Items) == 0 {
-		return nil, errors.Errorf("No mesh found in namespace [%s]", v.namespace)
+		return nil, errors.Errorf("No mesh found for namespace [%s]", v.namespace)
```

Comment:
"No mesh found in namespace [%s]" translates to there was no osm control plane installed **in** the namespace (the default is `osm-system`)

Reply:
Hi @jaellio will "No control plane for namespace [%s]" be more apt then?

Reply:
Hi @jaellio as per doc we add the namespace to OSM control plane, so I thought that for every OSM controlled namespace there is mesh(control plane) hence "mesh for namespace".
When I first saw the error message I was in doubt(mesh is added to namespace or namespace is added to mesh?) pardon me if I am missing something.